src/shared/bg_pmove.cpp

```diff
@@ -2011,12 +2011,12 @@ static void PM_WalkMove( pml_t & pml )
 	float scale = PM_CmdScale( &cmd, false );
 
 	// set the movementDir so clients can rotate the legs for strafing
-	PM_SetMovementDir();
+	PM_SetMovementDir(); // }
 
 	// project moves down to flat plane
 	pml.forward[ 2 ] = 0;
 	pml.right[ 2 ] = 0;
-
+	// dup {
```

Comment:
Huh?

Reply:
was a helper to allow me to better identify the stuff still to dedup, I moved it in another commit on top, so that it won't appear in code anymore (think: construction marks, nothing to do in real code indeed)